include/nbl/video/IGPUObjectFromAssetConverter.h

```diff
@@ -346,6 +447,23 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUBuffer** const _begin
     }
     finalizeBlock();
 
+    cmdbuf->end();
+    IGPUQueue::SSubmitInfo submit;
+    submit.commandBufferCount = 1u;
+    auto* cmdbuf_ptr = cmdbuf.get();
+    submit.commandBuffers = &cmdbuf_ptr;
+    // TODO have one signal semaphore here and write it into SParams
+    submit.signalSemaphoreCount = 0u;
+    submit.pSignalSemaphores = nullptr;
+    submit.waitSemaphoreCount = 0u;
+    submit.pWaitDstStageMask = nullptr;
+    submit.pWaitSemaphores = nullptr;
+    // dont event tell the queue to signal a fence after last submit if it'll never be touched by user
+    auto* signalFence = _params.transferFence ? fence.get() : nullptr;
+    _params.transferQueue->submit(1u, nullptr, signalFence);
```

Comment:
To be able to multithread, we need to probably provide an
```cpp
class CThreadSafeQueueAdapter : IQueue
{
     CThreadSafeQueueAdapter(IQueue* originalQueue);
};
```
basically a method forwarding class around an externally synchronised `IQueue` (make the COpenGLQueue unmutexed just how the Vulkan queue will be unsafe to touch from multiple threads), its something banal to write that could be tasked to @sadiuk